services/agents/registry.go

```diff
@@ -533,6 +553,109 @@ func (r *Registry) Collect(ch chan<- prom.Metric) {
 	r.mClockDrift.Collect(ch)
 }
 
+// StartPTSummaryAction starts pt-summary action on pmm-agent.
+//nolint:unparam
+func (r *Registry) StartPTSummaryAction(ctx context.Context, a *models.PtSummaryAction) error {
+	aRequest := &agentpb.StartActionRequest{
+		ActionId: a.ID,
+		Type:     managementpb.ActionType_PT_SUMMARY,
+		Params: &agentpb.StartActionRequest_ProcessParams_{
+			ProcessParams: &agentpb.StartActionRequest_ProcessParams{
+				Args: a.Args(),
+			},
+		},
+	}
+
+	agent, err := r.get(a.PMMAgentID)
+	if err != nil {
+		return errors.Wrap(err, "couldn't start action")
```

Comment:
`get()` already returns user-visible gRPC error. By wrapping it, we are trashing it.